src/services/schemas/SchemasService.ts

```diff
@@ -67,6 +67,7 @@ export class SchemaService {
         error => deferred.reject(error))
       .then(error => {
           if (isNotNullOrUndefined(error)) {
+            this.unregisterProtoSchema(container, schema.fileName);
```

Comment:
I don't think we need to retry. How many times should we retry? The failure of not being able to remove schema is not a big deal @ryanemerson